packages/kotti-ui/source/kotti-field/types.ts

```diff
@@ -20,40 +20,44 @@ export namespace KottiField {
 			 * Useful for checking validation on required fields
 			 */
 			isEmpty: (value: DATA_TYPE) => boolean
-			props: KottiField.Props<DATA_TYPE>
+			props: KottiField.Props<DATA_TYPE, PLACEHOLDER_TYPE>
 			supports: KottiField.Supports
 		}
 
-		export type Returns<DATA_TYPE> = {
-			currentValue: KottiField.Props<DATA_TYPE>['value']
-			helpDescription: KottiField.Props<DATA_TYPE>['helpDescription']
-			helpText: KottiField.Props<DATA_TYPE>['helpText']
-			hideClear: KottiField.Props<DATA_TYPE>['hideClear']
-			hideValidation: KottiField.Props<DATA_TYPE>['hideValidation']
+		export type Returns<DATA_TYPE, PLACEHOLDER_TYPE> = Pick<
+			KottiField.Props<DATA_TYPE, PLACEHOLDER_TYPE>,
+			| 'helpDescription'
+			| 'helpText'
+			| 'hideClear'
+			| 'hideValidation'
+			| 'isDisabled'
+			| 'isOptional'
+			| 'size'
+			| 'label'
+			| 'leftIcon'
+			| 'prefix'
+			| 'rightIcon'
+			| 'suffix'
+		> & {
+			currentValue: KottiField.Props<DATA_TYPE, PLACEHOLDER_TYPE>['value']
 			inputProps: Readonly<{
 				/**
 				 * Native HTML Props should have lowercase keys
 				 */
 				'data-test': string
 				disabled: boolean
-				tabindex: KottiField.Props<DATA_TYPE>['tabIndex']
+				tabindex: KottiField.Props<DATA_TYPE, PLACEHOLDER_TYPE>['tabIndex']
 			}>
-			isDisabled: KottiField.Props<DATA_TYPE>['isDisabled']
 			isEmpty: boolean
 			isLoading: KottiForm.Props['isLoading']
```

Comment:
we probably had this accidentally, can u make the type `KottiField.Props` thus move it up